File: python-programming/examples/numpy/fibonacci.py
import numpy as np

def fibo(n):
    if n == 1 or n == 2:
        return 1
    a, b = 1, 1
    for _ in range(n-2):
        a, b = b, a + b
    return b

vfibo = np.vectorize(fibo)
a = np.array([
    [1, 2, 3, 4, 5, 6],
    [7, 8, 9, 10, 11, 12],
    ])
print(a)
print(a.dtype)
print()

b = vfibo(a)
print(b)
print(b.dtype)

File: javascript/examples/functions/fibonacci.js
export default function fibo(n) {
	if (n == 1) {
		return [1];
	}
	if (n == 2) {
		return [1, 1];
	}
	let numbers = [1]
	for (let c = 3; c <= n; c++) {
		numbers.push(numbers[-1] + numbers[-2]);
	}
	return numbers;
	
}